qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/codeactions/CodeActionRequest.java

```diff
@@ -87,6 +87,22 @@ public Node getCoveredNode() throws BadLocationException {
 
 	}
 
+	/**
+	 * Returns the covered node given by the offset and null otherwise.
+	 *
+	 * @return Returns the covered node given by the offset and null otherwise.
+	 *
+	 * @throws BadLocationException
+	 */
+	public Node getCoveredNode(int offset) throws BadLocationException {

```

Comment:
Remove this method and update CodeActionRequest to extends AbstractPositionRequest